climada/hazard/relative_cropyield.py

```diff
@@ -495,7 +504,7 @@ def init_full_hazard_set(input_dir=INPUT_DIR, output_dir=OUTPUT_DIR, bbox=BBOX,
         # historic file
         crop_irr = (((items[i])[6]).split('-'))[1] + '-' + (((items[i])[6]).split('-'))[2]
 
-        cp_his = CropPotential()
+        cp_his = RelativeCropyield()
         cp_his.set_from_single_run(input_dir=input_dir, bbox=bbox, yearrange=yearrange,
                                    ag_model=(items[i])[0], cl_model=(items[i])[1],
                                    scenario=(items[i])[3], soc=(items[i])[4],
```

Comment:
Is it possible to reduce the number of hard-coded list indexing (in all the lines below)? The hardcoded indices are completely obscure and are not easy to handle in future changes of the code.

Reply:
An article on hardcoded index.

https://treyhunner.com/2018/03/tuple-unpacking-improves-python-code-readability/